packages/wonder-stuff-ci/src/exec-prom.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * A simple promisified version of child_process.exec, so we can `await` it
+ */
+import {exec, ExecOptions} from "child_process";
+import util from "util";
+
+export const bufferToString = (input: Buffer | string): string => {
```

Comment:
**suggestion:** Generally, we keep each file to one export. It makes discoverability easier and we have no quota on files so there's little reason not to do it. As it stands, it's kinda confusing that the `exex-prom.ts` file exports `bufferToString`. Can you move `bufferToString` to `buffer-to-string.ts`?